author	Syam Sidhardhan <s.syam@samsung.com>
commit	9ec213277b46867fca20aae85ca3f76cec5ba56d
	Fri, 13 Apr 2012 09:36:41 +0000 (12:36 +0300)

gstavdtpsink: Fix D-Bus memory leak in audio

diff -cr bluez-4.99.pan.orig/audio/gstavdtpsink.c bluez-4.99.pan/audio/gstavdtpsink.c
*** bluez-4.99.pan.orig/audio/gstavdtpsink.c	Wed Dec 21 17:53:54 2011
--- bluez-4.99.pan/audio/gstavdtpsink.c	Mon Oct 15 04:17:47 2012
***************
*** 1336,1341 ****
--- 1336,1343 ----
  	reply = dbus_connection_send_with_reply_and_block(self->data->conn,
  							msg, -1, &err);
  
+ 	dbus_message_unref(msg);
+ 
  	if (dbus_error_is_set(&err))
  		goto fail;
  
***************
*** 1362,1368 ****
  	dbus_error_free(&err);
  
  	if (reply)
! 		dbus_message_unref(msg);
  
  	return FALSE;
  }
--- 1364,1370 ----
  	dbus_error_free(&err);
  
  	if (reply)
! 		dbus_message_unref(reply);
  
  	return FALSE;
  }
